library/std/src/env.rs

```diff
@@ -695,21 +695,21 @@ pub struct ArgsOs {
     inner: sys::args::Args,
 }
 
-/// Returns the arguments which this program was started with (normally passed
```

Comment:
I think which is correct. Is it not?

Reply:
I believe “that” is correct because this is a defining clause.